### PR TITLE
[FIX] skip tests that do not work with root rights

### PR DESCRIPTION
**Pull request by eseiler**
Follow-up should be a cron that runs ctest with sudo :)

### CODE REVIEW

**Review by SGSSGene — APPROVED** [review]

File: test/unit/test/CMakeLists.txt [hunk 1]
```diff
@@ -1,8 +1,9 @@
 seqan3_test(expect_range_eq_test.cpp)
 seqan3_test(expect_same_type_test.cpp)
+seqan3_test(permissions.cpp)
```
Comment (marehr) [inline test/unit/test/CMakeLists.txt:3]:
```suggestion
seqan3_test(file_access_test.cpp)
```

?

File: test/unit/test/permissions.cpp [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+// Copyright (c) 2016-2021, Knut Reinert & MPI für molekulare Genetik
+// This file may be used, modified and/or redistributed under the terms of the 3-clause BSD-License
+// shipped with this file and also available at: https://github.com/seqan/seqan3/blob/master/LICENSE.md
+// -----------------------------------------------------------------------------------------------------
+
+#include <gtest/gtest.h>
+
+#include <seqan3/test/file_access.hpp>
+#include <seqan3/test/tmp_directory.hpp>
+#include <seqan3/test/tmp_filename.hpp>
```
Comment (marehr) [inline test/unit/test/permissions.cpp:12]: This test is cyclic;

* seqan3/test/file_access.hpp
* tested by file_access_test.cpp
* uses seqan3/test/tmp_directory.hpp
* tested by tmp_directory_test.cpp
* uses seqan3/test/file_access.hpp

Can we write this test in a way that does not need seqan3/test/tmp_directory.hpp and seqan3/test/tmp_filename.hpp?

**Review by marehr — CHANGES_REQUESTED** [review]

File: test/unit/test/permissions.cpp (hunk 2 above)
Comment (eseiler) [reply test/unit/test/permissions.cpp:12]: I could, e.g., use the current working directory and skip the tests if I do not have permissions to write there

Comment (marehr) [reply test/unit/test/permissions.cpp:12]: Sounds like a great idea.

**Review by marehr — APPROVED** [review]
noice! ty!

**Pull request merged**